lib/Reader.php

```diff
@@ -105,7 +105,7 @@ public function parse(): array
      *
      * @param array<string, mixed>|null $elementMap
      *
-     * @return array<string, mixed>
+     * @return array<int,array<string, mixed>>
```

Comment:
This was an existing error in the PHPdoc. This thing returns an array of arrays.